beetsplug/lastgenre/__init__.py

```diff
@@ -584,11 +608,12 @@ def _tags_for(self, obj, min_weight=None):
         # Work around an inconsistency in pylast where
         # Album.get_top_tags() does not return TopItem instances.
         # https://github.com/pylast/pylast/issues/86
+        obj_to_query: Any = obj
         if isinstance(obj, pylast.Album):
-            obj = super(pylast.Album, obj)
+            obj_to_query = super(pylast.Album, obj)
```

Comment:
just fyi @henry-oberholtzer: here a new variable was the solution to not changing the type along the way which mypy does not like